app/Http/Controllers/TrainingDash.php

```diff
@@ -341,7 +341,7 @@ public function saveNewTicket(Request $request) {
         ];
 
         $client = new Client();
-        $res = $client->request('POST', 'https://api.vatusa.net/v2/user/' . $request->controller . '/training/record?apikey=' . Config::get('vatusa.api_key'), $req_params);
+        $res = $client->request('POST', Config::get('vatusa.base').'/v2/user/' . $request->controller . '/training/record?apikey=' . Config::get('vatusa.api_key'), $req_params);
```

Comment:
See comment above - move to separate issue.

Reply:
Removed